### PR TITLE
Jupiter 5.9.0 + Vintage 5.9.0 + Platform 1.9.0

### PR DESCRIPTION
**Pull request by LuciferYang**
To fix https://github.com/sbt/sbt-jupiter-interface/issues/52, upgrade to support Junit 5.9.0

### CODE REVIEW

File: src/library/src/test/java/net/aichler/jupiter/tickets/n0001/Ticket0001Test.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public void test() {
 
         assertThat(testRunner.eventHandler().byStatus(Status.Failure), hasSize(1));
         assertThat(testRunner.logger().byLevel("error"), hasItem(
-                containsString("Could not find method [dataProvider]")
+                containsString("Could not find factory method [dataProvider]")
```
Comment (LuciferYang) [inline src/library/src/test/java/net/aichler/jupiter/tickets/n0001/Ticket0001Test.java:25]: the error message changed due to 

https://github.com/junit-team/junit5/commit/feeb5bcb6962c7ef997c04ecba474bb17f01480d

**Comment by LuciferYang** [comment]
CI passed

https://github.com/LuciferYang/sbt-jupiter-interface/runs/7593353074?check_suite_focus=true

image

**Review by maichler — APPROVED** [review]
Thank you very much for your contribution :+1:

**Comment by LuciferYang** [comment]
thanks @maichler

**Pull request merged**